cloudify_cli/bootstrap/bootstrap.py

```diff
@@ -129,6 +130,15 @@ def validate_manager_deployment_size(blueprint_path):
                                     MAX_MANAGER_DEPLOYMENT_SIZE))
 
 
+def _validate_credentials_are_set():
+    cred_env_vars = (CLOUDIFY_USERNAME_ENV_VAR, CLOUDIFY_PASSWORD_ENV_VAR)
+    if not set(cred_env_vars).issubset(os.environ.keys()):
+        raise CloudifyBootstrapError(
+            'The following environment variables must be set before '
+            'bootstrapping a secured manager: {0}'.
+            format(', '.join(cred_env_vars)))
```

Comment:
we tend to place the dot preceding the format and not at the end of the previous line